build_library/build_image_util.sh

```diff
@@ -273,6 +273,58 @@ write_packages() {
     image_packages "$1" | sort > "$2"
 }
 
+get_metadata() {
+    local prefix="$1"
+    local pkg="$2"
+    local key="$3"
+    local path="${prefix}/var/db/pkg/${pkg%%:*}/${key}"
+    local val
+    if [[ -f "$path" ]]; then
+        val="$(< $path)"
```

Comment:
What is val supposed to be? Could you give it a more meaningful name?

Reply:
There is `key` and `val`. I can write `value` but I thought it was a common abbreviation :)

Reply:
Ok, I didn't realize that this was the value associated with the key... I guess I just find this "expose key-value pairs as files" rather confusing... Anyway, I guess it's fine.